trusted-profiles/README.md

```diff
@@ -141,7 +141,7 @@ TRUSTED_PROFILE_NAME=code-engine-cos-access
 ibmcloud ce job create --name ${JOB_NAME} \
   --build-source https://github.com/IBM/CodeEngine \
   --build-context-dir trusted-profiles/${PROGRAMMING_LANGUAGE} \
-  --trusted-profiles-enabled true \
+  --trusted-profiles-enabled=true \
```

Comment:
That is interesting. Is it by intention?

Reply:
Yes, it has to be there. Boolean flags require the `=` sign in the Go CLI libraries. Without it, it does not work.